client/src/api.js

```diff
@@ -44,6 +23,11 @@ const API = {
       params.headers[authHeaderName] = authHeaderValue
     }
 
+    const authHeader = API._getAuthHeader()
+    if (authHeader) {
+      params.headers[authHeader[0]] = authHeader[1]
+    }
+
```

Comment:
Remove the 5 lines above these!

Reply:
I think you removed the wrong 5 lines (you removed the ones I quoted); the ones you left in are:
```
    const [authHeaderName, authHeaderValue] = API._getAuthHeader()
    if (authHeaderName && authHeaderValue) {
      params.headers[authHeaderName] = authHeaderValue
    }

```
This can fail because `API._getAuthHeader()` was changed to `return null` and you can't destructure `null`…
So reinstate the 'new' lines and remove the 'old' ones.